contentcuration/contentcuration/frontend/shared/views/MainNavigationDrawer.vue

```diff
@@ -73,7 +76,11 @@
         </VListTile>
       </VList>
       <VContainer>
-        <KolibriLogo :height="75" />
+        <KLogo
+          altText="Kolibri logo"
```

Comment:
Seems like this alt text should be translated. Although for the rebranding release, I think we're trying to avoid translating things. It also sucks I don't think we can have `KLogo` just have this internally.